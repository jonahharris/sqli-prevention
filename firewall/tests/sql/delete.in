>>> Sada testovacich SQL dotazu: delete
# drop table
DROP TABLE Employees, Customers;
DROP TEMPORARY TABLE authors;
DROP TABLE IF EXISTS t1;
DROP TABLE IF EXISTS t1, t2, t3 WAIT 14;
DROP TABLE IF EXISTS t1, t2 NOWAIT;
DROP TABLE IF EXISTS t1, t2 RESTRICT;
DROP TABLE IF EXISTS t1, t2 CASCADE;
DROP TABLE IF EXISTS t1, t2, t3 WAIT 14 CASCADE;
# delete table
DELETE FROM page_hit ORDER BY timestmp LIMIT 1000000;
DELETE FROM t;
DELETE post FROM blog INNER JOIN post WHERE blog.id = post.blog_id;
DELETE FROM MyGuests WHERE id=3;
# Truncate
truncate authors;
truncate table authors;
truncate table authors nowait;
truncate table authors wait 5;
truncate authors nowait;
truncate authors wait 5;
