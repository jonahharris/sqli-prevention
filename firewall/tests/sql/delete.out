>>> Sada testovacich SQL dotazu: delete
# drop table
TABLE Employees;TABLE Customers;DROPTABLE 0 0 2 0;STMT;1;0;
TABLE authors;DROPTABLE 1 0 1 0;STMT;1;0;
TABLE t1;DROPTABLE 0 1 0 0;STMT;1;0;
TABLE t1;TABLE t2;TABLE t3;DROPTABLE 0 1 14 0;STMT;1;0;
TABLE t1;TABLE t2;DROPTABLE 0 1 0 0;STMT;1;0;
TABLE t1;TABLE t2;DROPTABLE 0 1 0 1;STMT;1;0;
TABLE t1;TABLE t2;DROPTABLE 0 1 0 2;STMT;1;0;
TABLE t1;TABLE t2;TABLE t3;DROPTABLE 0 1 14 2;STMT;1;0;
# delete table
NAME timestmp;ORDERBY 0;ORDERBYLIST 1;NUMBER 1000000;LIMIT 1;TABLE page_hit;DELETE ONE 0;STMT;1;0;
TABLE t;DELETE ONE 0;STMT;1;0;
TABLE post;TABLE blog;TABLE post;JOIN 101;FIELDNAME blog.id;FIELDNAME post.blog_id;CMP 4;WHERE;DELETE MULTI 0 1 1;STMT;1;0;
NAME id;NUMBER 3;CMP 4;WHERE;TABLE MyGuests;DELETE ONE 0;STMT;1;0;
# Truncate
TABLE authors;TRUNCATE 0;STMT;1;0;
TABLE authors;TRUNCATE 0;STMT;1;0;
TABLE authors;TRUNCATE 0;STMT;1;0;
TABLE authors;TRUNCATE 5;STMT;1;0;
TABLE authors;TRUNCATE 0;STMT;1;0;
TABLE authors;TRUNCATE 5;STMT;1;0;
